src/runtime_src/core/tools/xbutil2/SubCmdValidate.cpp

```diff
@@ -1673,10 +1728,28 @@ SubCmdValidate::execute(const SubCmdOptions& _options) const
         xclbin_location.append("/");
     }
 
+    //check if param option is provided
+    if (!sParam.empty()) {
+      XBU::verbose("Sub command: --param");
+      boost::split(param, sParam, boost::is_any_of(":")); // eg: dma:block-size:1024
+
+      //check parameter format
+      if (param.size() != 3)
+        throw xrt_core::error((boost::format("Invalid parameter format : '%s'") % sParam).str());
+      
+      //check test case name
+      doesTestExists(param[0], testNameDescription);
+
+      //check parameter name
+      std::vector<std::string> valid_param_list = {"block-size"};
```

Comment:
This data should be defined only in 1 place.  Earlier, there was a collection that described the description of the parameter and the DMA test it was associated to.  This collection should be expanded to be a vector of structures where the structure contains:
```
{
   std::string testName;
   std::string paramName;
   std::string description;
}
```
And used here to determine if the "validation test" supports the given param.